src/js/tracks/text-track-list-converter.js

```diff
@@ -0,0 +1,47 @@
+let ttToJson = function(track) {
+  return {
+    kind: track.kind,
+    label: track.label,
+    language: track.language,
+    id: track.id,
+    inBandMetadataTrackDispatchType: track.inBandMetadataTrackDispatchType,
+    mode: track.mode,
+    cues: track.cues && Array.prototype.map.call(track.cues, function(cue) {
+      return {
+        startTime: cue.startTime,
+        endTime: cue.endTime,
+        text: cue.text,
+        id: cue.id
+      };
+    }),
+    src: track.src
+  };
+};
+
+let textTracksToJson = function(tech) {
```

Comment:
The naming of `textTracksToJson` compared to `ttToJson ` could be a little more different.